### ISSUE TITLE
How to deal with unknown height

### ISSUE THREAD

**Issue by leolux** [dhorions/boxable#207]
How to deal with text of unknown length resulting in a table row of unknown height?

**Comment by Frulenzo** [comment]
Yeah, that's a problem. You always need to look after that your table row height is not bigger than A4 page. You need to find some "sweet spot" with number of charachters and break your text into two rows.

**Comment by quodlibetbv** [comment]
This issue no longer exists, see unit tests CellOverflowAcrossPagesTest and LargeRowIssueTest and ConsecutiveTablesmultiPageOverlapTest.

If you have a code sample that still shows the issue with latest version (1.8.2 or main branch), please reopen.
